### DISCUSSION TITLE
Unable to pair end devices via Sonoff S26R2ZB as a router

### DISCUSSION THREAD

**Discussion by cwilko** [arendst/Tasmota#15125] [Q&A] [answered]
### PROBLEM DESCRIPTION

I have several end devices which work fine when connected to my coordinator (Sonoff Zigbee Bridge + Tasmota). 
However, when I pair a device in close proximity to a Sonoff S26R2ZB - to use it as a router - the pairing connection does not complete. 

I receive the following message when attempting to pair:

`22:54:49.625 MQT: tele/tasmota/RESULT = {"ZbState":{"Status":34,"IEEEAddr":"0x00124B0024FC3813","ShortAddr":"0x6829","ParentNetwork":"0x9370","JoinStatus":1,"Decision":0}}
`
...and then nothing. No further messages in the log, no telemetry messages received from the device. 
A ZbMap scan shows that the end device is a child of the S26R2ZB but the full set of pairing output is not received, and no telemetry is received from any end devices. 

For comparison, a successful pair to the co-ordinator looks like this:

```
22:54:04.913 MQT: tele/tasmota/RESULT = {"ZbState":{"Status":34,"IEEEAddr":"0x00124B0024FC3813","ShortAddr":"0x0BAB","ParentNetwork":"0x0000","JoinStatus":1,"Decision":0}}
22:54:05.770 MQT: tele/tasmota/RESULT = {"ZbState":{"Status":30,"IEEEAddr":"0x00124B0024FC3813","ShortAddr":"0x0BAB","PowerSource":false,"ReceiveWhenIdle":false,"Security":false}}
22:54:06.936 MQT: tele/tasmota/RESULT = {"ZbState":{"Status":32,"ActiveEndpoints":["0x01"]}}
22:54:07.482 MQT: tele/tasmota/SENSOR = {"ZbReceived":{"0x0BAB":{"Device":"0x0BAB","Name":"Device2","ModelId":"TH01","Manufacturer":"eWeLink","Endpoint":1,"LinkQuality":61}}}
22:54:09.031 MQT: tele/tasmota/RESULT = {"ZbState":{"Status":33,"Device":"0x0BAB","Endpoint":"0x01","ProfileId":"0x0104","DeviceId":"0x0302","DeviceVersion":0,"InClusters":["0x0000","0x0003","0x0402","0x0405","0x0001"],"OutClusters":["0x0003"]}}
22:54:11.029 ZIG: auto-bind `ZbBind {"Device":"0x0BAB","Endpoint":1,"Cluster":"0x0001"}`
22:54:11.572 MQT: tele/tasmota/RESULT = {"ZbBind":{"Device":"0x0BAB","Name":"Device2","Status":0,"StatusMessage":"SUCCESS"}}
22:54:13.071 ZIG: auto-bind `ZbBind {"Device":"0x0BAB","Endpoint":1,"Cluster":"0x0402"}`
22:54:13.337 MQT: tele/tasmota/RESULT = {"ZbBind":{"Device":"0x0BAB","Name":"Device2","Status":0,"StatusMessage":"SUCCESS"}}
22:54:15.037 ZIG: auto-bind `ZbBind {"Device":"0x0BAB","Endpoint":1,"Cluster":"0x0405"}`
22:54:15.941 MQT: tele/tasmota/RESULT = {"ZbBind":{"Device":"0x0BAB","Name":"Device2","Status":0,"StatusMessage":"SUCCESS"}}
22:54:17.041 ZIG: auto-bind `ZbSend {"Device":"0x0BAB","Config":{"BatteryVoltage":{"MinInterval":3600,"MaxInterval":14400,"ReportableChange":0.2},"BatteryPercentage":{"MinInterval":3600,"MaxInterval":14400,"ReportableChange":5}}}`
22:54:17.563 MQT: tele/tasmota/SENSOR = {"ZbReceived":{"0x0BAB":{"Device":"0x0BAB","Name":"Device2","ConfigResponse":{},"Endpoint":1,"LinkQuality":61}}}
22:54:19.061 ZIG: auto-bind `ZbSend {"Device":"0x0BAB","Config":{"Temperature":{"MinInterval":30,"MaxInterval":3600,"ReportableChange":0.5}}}`
22:54:20.120 MQT: tele/tasmota/SENSOR = {"ZbReceived":{"0x0BAB":{"Device":"0x0BAB","Name":"Device2","ConfigResponse":{},"Endpoint":1,"LinkQuality":53}}}
22:54:21.069 ZIG: auto-bind `ZbSend {"Device":"0x0BAB","Config":{"Humidity":{"MinInterval":30,"MaxInterval":3600,"ReportableChange":1}}}`
22:54:21.678 MQT: tele/tasmota/SENSOR = {"ZbReceived":{"0x0BAB":{"Device":"0x0BAB","Name":"Device2","ConfigResponse":{},"Endpoint":1,"LinkQuality":55}}}
22:54:26.499 MQT: tele/tasmota/SENSOR = {"ZbReceived":{"0x0BAB":{"Device":"0x0BAB","Name":"Device2","Temperature":19.72,"Humidity":68.21,"Endpoint":1,"LinkQuality":55}}}
```
I can successfully pair directly to the co-ordinator, and also via other types of router, such as the Hive Light. 

### REQUESTED INFORMATION
_Make sure your have performed every step and checked the applicable boxes before submitting your issue. Thank you!_

- [] Read the [Contributing Guide and Policy](https://github.com/arendst/Tasmota/blob/development/CONTRIBUTING.md) and [the Code of Conduct](https://github.com/arendst/Tasmota/blob/development/CODE_OF_CONDUCT.md)
- [] Searched the problem in [issues](https://github.com/arendst/Tasmota/issues)
- [x] Searched the problem in [discussions](https://github.com/arendst/Tasmota/discussions)
- [x] Searched the problem in the [docs](https://tasmota.github.io/docs/FAQ)
- [x] Searched the problem in the [chat](https://discord.gg/Ks2Kzd4)
- [x] Device used (e.g., Sonoff Basic): Sonoff S26R2ZB
- [x] Tasmota binary firmware version number used: 11.0.0(zbbridge)
  - [x] Pre-compiled
  - [ ] Self-compiled
- [x] Flashing tools used: Serial-to-USB
- [x] Provide the output of command: `Backlog Template; Module; GPIO 255`:

22:59:53.541 MQT: stat/tasmota/RESULT = {"NAME":"Generic","GPIO":[1,1,1,1,1,1,1,1,1,1,1,1,1,1],"FLAG":0,"BASE":18}
22:59:53.778 MQT: stat/tasmota/RESULT = {"Module":{"75":"Sonoff ZbBridge"}}
22:59:53.982 MQT: stat/tasmota/RESULT = {"GPIO0":{"320":"Led_i1"},"GPIO1":{"3552":"Zigbee Tx"},"GPIO2":{"0":"None"},"GPIO3":{"3584":"Zigbee Rx"},"GPIO4":{"5312":"Zigbee Rst1"},"GPIO5":{"0":"None"},"GPIO9":{"0":"None"},"GPIO10":{"0":"None"},"GPIO12":{"640":"I2C SDA"},"GPIO13":{"576":"LedLink_i"},"GPIO14":{"608":"I2C SCL"},"GPIO15":{"0":"None"},"GPIO16":{"32":"Button1"},"GPIO17":{"0":"None"}}

```lua
  Configuration output here:


```
- [x] If using rules, provide the output of this command: `Backlog Rule1; Rule2; Rule3`:
```lua
  Rules output here:


```
- [x] Provide the output of this command: `Status 0`:
```lua
  STATUS 0 output here:
23:00:48.719 MQT: stat/tasmota/STATUS = {"Status":{"Module":75,"DeviceName":"Tasmota","FriendlyName":["Tasmota"],"Topic":"tasmota","ButtonTopic":"0","Power":0,"PowerOnState":3,"LedState":1,"LedMask":"FFFF","SaveData":1,"SaveState":1,"SwitchTopic":"0","SwitchMode":[0,0,0,0,0,0,0,0],"ButtonRetain":0,"SwitchRetain":0,"SensorRetain":0,"PowerRetain":0,"InfoRetain":0,"StateRetain":0}}
23:00:48.725 MQT: stat/tasmota/STATUS1 = {"StatusPRM":{"Baudrate":115200,"SerialConfig":"8N1","GroupTopic":"tasmotas","OtaUrl":"http://ota.tasmota.com/tasmota/release/tasmota.bin.gz","RestartReason":"Power On","Uptime":"0T09:37:05","StartupUTC":"2022-03-12T12:23:43","Sleep":50,"CfgHolder":4617,"BootCount":15,"BCResetTime":"2022-03-03T16:49:33","SaveCount":28,"SaveAddress":"FA000"}}
23:00:48.730 MQT: stat/tasmota/STATUS2 = {"StatusFWR":{"Version":"11.0.0(zbbridge)","BuildDateTime":"2022-02-12T14:13:46","Boot":31,"Core":"2_7_4_9","SDK":"2.2.2-dev(38a443e)","CpuFrequency":160,"Hardware":"ESP8266EX","CR":"369/699"}}
23:00:48.734 MQT: stat/tasmota/STATUS3 = {"StatusLOG":{"SerialLog":0,"WebLog":2,"MqttLog":2,"SysLog":2,"LogHost":"","LogPort":514,"SSId":["SquirrelCottage",""],"TelePeriod":300,"Resolution":"558180C0","SetOption":["00008009","2805C80001000600003C5A00190000000000","00000080","00006000","00004000"]}}
23:00:48.743 MQT: stat/tasmota/STATUS4 = {"StatusMEM":{"ProgramSize":673,"Free":1116,"Heap":23,"ProgramFlashSize":2048,"FlashSize":2048,"FlashChipId":"1540A1","FlashFrequency":40,"FlashMode":3,"Features":["00000809","0F1007C6","04400001","00000003","00000000","00000000","00020080","00200000","00000000"],"Drivers":"1,2,4,7,9,10,12,20,23,38,41,50","Sensors":"1"}}
23:00:48.749 MQT: stat/tasmota/STATUS5 = {"StatusNET":{"Hostname":"tasmota-7975","IPAddress":"192.168.1.55","Gateway":"192.168.1.27","Subnetmask":"255.255.255.0","DNSServer1":"8.8.8.8","DNSServer2":"8.8.4.4","Mac":"48:3F:DA:2A:9F:27","Webserver":2,"HTTP_API":1,"WifiConfig":4,"WifiPower":17.0}}
23:00:48.753 MQT: stat/tasmota/STATUS6 = {"StatusMQT":{"MqttHost":"192.168.1.202","MqttPort":1883,"MqttClientMask":"DVES_%06X","MqttClient":"DVES_2A9F27","MqttUser":"DVES_USER","MqttCount":3,"MAX_PACKET_SIZE":1200,"KEEPALIVE":30,"SOCKET_TIMEOUT":4}}
23:00:48.759 MQT: stat/tasmota/STATUS7 = {"StatusTIM":{"UTC":"2022-03-12T22:00:48","Local":"2022-03-12T23:00:48","StartDST":"2022-03-27T02:00:00","EndDST":"2022-10-30T03:00:00","Timezone":"+01:00","Sunrise":"07:09","Sunset":"18:49"}}
23:00:48.765 MQT: stat/tasmota/STATUS10 = {"StatusSNS":{"Time":"2022-03-12T23:00:48"}}
23:00:48.771 MQT: stat/tasmota/STATUS11 = {"StatusSTS":{"Time":"2022-03-12T23:00:48","Uptime":"0T09:37:05","UptimeSec":34625,"Vcc":3.550,"Heap":23,"SleepMode":"Dynamic","Sleep":50,"LoadAvg":19,"MqttCount":3,"Wifi":{"AP":1,"SSId":"SquirrelCottage","BSSId":"60:E3:27:DB:86:19","Channel":1,"Mode":"11n","RSSI":68,"Signal":-66,"LinkCount":3,"Downtime":"0T00:00:48"}}}

```
- [x] Set `weblog` to 4 and then, when you experience your issue, provide the output of the Console log:
```lua
  Console output here:
22:54:49.625 MQT: tele/tasmota/RESULT = {"ZbState":{"Status":34,"IEEEAddr":"0x00124B0024FC3813","ShortAddr":"0x6829","ParentNetwork":"0x9370","JoinStatus":1,"Decision":0}}


```

### TO REPRODUCE
_Steps to reproduce the behavior:_
Attempt to pair a device via Sonoff S26R2ZB as a router

### EXPECTED BEHAVIOUR
_A clear and concise description of what you expected to happen._
Pair successfully and traffic received from end devices

### SCREENSHOTS
_If applicable, add screenshots to help explain your problem._

### ADDITIONAL CONTEXT
_Add any other context about the problem here._

**(Please, remember to close the issue when the problem has been addressed)**

**Comment by MattWestb** [comment, marked as answer]
More then likely firmware problem if reading all problem Z2M is having with this devices https://github.com/Koenkk/zigbee2mqtt/issues/10282 as only one example.
Hopefully SoOff is releasing one updated firmware for it.

Also is the radio antenna is bad placed and is partly grounded from the relay so its not working OK 360°.
